### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by a-guiducci**
updating changelog for 4.0.0 release

### CODE REVIEW

**Review by bryanbellsmith — CHANGES_REQUESTED** [review]
I feel that this should be a new release, rather than modifying the existing `4.0.0-alpha` release. Also, I believe we have to update `src/privateai_client/__about__.py` to the correct version.

**Comment by a-guiducci** [comment]
> I feel that this should be a new release, rather than modifying the existing `4.0.0-alpha` release. Also, I believe we have to update `src/privateai_client/__about__.py` to the correct version.

we never pushed a 4.0 alpha release to pypi so I would argue against that

**Comment by a-guiducci** [comment]
> > > I feel that this should be a new release, rather than modifying the existing `4.0.0-alpha` release. Also, I believe we have to update `src/privateai_client/__about__.py` to the correct version.
> > 
> > 
> > we never pushed a 4.0 alpha release to pypi so I would argue against that
> 
> We did, in fact: https://pypi.org/project/privateai-client/#history

ah, well nvm then!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,20 @@
+[Unreleased]
```
Comment (bryanbellsmith) [inline CHANGELOG.md:1]: Minor nitpick
```suggestion
## [Unreleased]
```

**Review by bryanbellsmith — APPROVED** [review]
One minor nitpick, not blocking

**Pull request merged**